packages/strapi-plugin-content-manager/admin/src/components/Filter/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ *
+ * Filter
+ */
+
+
+import React from 'react';
+import moment from 'moment';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { get, toString, upperFirst } from 'lodash';
+import Flex from './Flex';
+import Remove from './Remove';
+import Separator from './Separator';
+
+
+function Filter({ filter, index, onClick, onClickOpen, schema }) {
+  let value = filter.value;
+
+  if (get(schema, [filter.attr, 'type']) === 'date') {
+    const format = filter.value
```

Comment:
You can do it in a better way using Moment.js (https://stackoverflow.com/a/26700097/5856363).

Reply:
Maybe dayjs is the better option, https://github.com/iamkun/dayjs